### PR TITLE
Styling/loading state

### LINKED ISSUE #47: Align active styles in Navigation

### LINKED ISSUE #48: Add background

### LINKED ISSUE #49: Remove Albums header from AlbumCardsDisplay

### LINKED ISSUE #50: Import a font

### LINKED ISSUE #51: Manage the loading sate

### PR DESCRIPTION
**Pull request by monshan**
# Selector FE PR

## What does this merge do? Check all that apply.
- [ ] Adds a new feature
- [x] Improves an existing feature
- [ ] Fixes a bug
- [ ] Cleans up (logs, tests, etc.)

## Summary of changes

- Imports video/mp4 assets and fonts
- Applies a potential video/mp4 background to the entire application
- Adjusts spacing on the header grid-area for the App component
- Removes header from the AlbumCardContainer
- Sets a loading placeholder for the Album card by importing the Skeleton component from @material-ui/lab

## Notes for reviewer (any known bugs, areas of interest, etc.)

- **New dependency added, be sure to run <code> npm install</code> again on branch checkout**
- The primary purpose of this branch is to add a loading state for the AlbumCard component that will serve as a placeholder until a successful response from the GraphQL server is received
- Some slight styling enhancements were also made (detailed above) and further assets were added to public in order to give some options for the application background

## Next Steps (Future Iterations)

- Ensure styling changes are in line with team members common styling changes

### CODE REVIEW

File: src/Components/AlbumCard/AlbumCard.js [hunk 1]
```diff
@@ -9,7 +10,15 @@ function AlbumCard ({ title }) {
     variables: { title: title }
   })
   
-  if (loading) return 'Loading...'
+  if (loading) return (
```
Comment (Max9545) [inline src/Components/AlbumCard/AlbumCard.js:13]: Good place for an animation at some point

File: src/Components/Search/Search.css [hunk 2]
```diff
@@ -3,11 +3,12 @@
   justify-content: flex-end;
   align-items: flex-end;
   margin-right: 1rem;
+  height: 100px;
 }
 
 .search__input {
```
Comment (Max9545) [inline src/Components/Search/Search.css:9]: The Navbar covers the 'Selector' logo header a little bit once it reaches mobil view size, make a little smaller or add margin possibly

File: src/Components/AlbumCard/AlbumCard.js [hunk 3]
```diff
@@ -9,7 +10,15 @@ function AlbumCard ({ title }) {
...
+    <div className='card_card' data-cy='card_card'>
+      <Skeleton animation="wave" variant="rect" width={250} height={250} />
```
Comment (rdtho2525) [inline src/Components/AlbumCard/AlbumCard.js:15]: These `Skeleton` components are really handy - I'll have to use these in the future.

**Review by rdtho2525 — APPROVED** [review]
+ `Skeleton` components render loading state as expected
+ `red-with-needle.mp4` has a strong contrast ratio with the foreground color scheme
+ All elements are mobile responsive
+ Fonts are attention-grabbing and easy to read
+ Future iterations (on another branch): 
  + Search bar styling
  + Resolve header/nav/search overlap

**Pull request merged**